README.md

```diff
@@ -132,6 +132,45 @@ Counter component.
 Note, to avoid errors, when adding a comment to a component template use the
 Jinja syntax, `{# comment here #}`, rather than the HTML syntax.
 
+### Pass data to a component
+
+You can, of course, pass data to your meld component. Meld is passing **kwargs 
+to the render function of the *meld* templatetag, so you can pass any number of 
+named arguments. The component is found based on the first parameter, aka name 
+of the component and any number of data passed afterwards. 
+
+Providing a very basic component as an example to display a greeting message using
+the passed value for the keyword "name" or the fallback "Nobody" in its template.
```

Comment:
The example doubles up on the fallback (it's being set to "Nobody" in the template and in the component, let's remove one of those)